cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -370,32 +370,28 @@ def test_compute_launch_template_properties(
 class LoginNodeLTAssertion:
     def __init__(
         self,
-        pool_name,
-        instance_type,
-        count,
         subnet_ids,
-        key_name,
-        security_groups,
-        gracetime_period,
+        root_volume,
+        image_id,
+        http_tokens,
+        iam_instance_profile_name,
     ):
-        self.pool_name = pool_name
-        self.instance_type = instance_type
-        self.count = count
         self.subnet_ids = subnet_ids
-        self.key_name = key_name
-        self.security_groups = security_groups
-        self.gracetime_period = gracetime_period
+        self.root_volume = root_volume
```

Comment:
This test is meant to confirm that the login node's root volume is encrpyted by default. Should we be passing the settings explicitly? Or can we simply go ahead and assert that the Ebs volume has `"Encrypted": True` in the launch template?